src/main/webapp/dev/js/common/instructorFeedbackResults.es6

```diff
@@ -319,6 +319,12 @@ function prepareInstructorFeedbackResultsPage() {
         });
     };
 
+    // decrease top-margin of expand button, if status message is present
+    if ($('#statusMessagesToUser').html().search('div') > 0) {
+        $('#expand-collapse-button').css('margin-top', '5px');
+        $('#expand-collapse-button + br').remove();
+    }
```

Comment:
> I feel the top margin can be reduced a bit, since it looks quite uneven.

I'm okay with the reduced top margin as long as it's done using pure CSS.